### ISSUE TITLE
train_vae should dump an image at each epoch, and train_{vae,nn} should run `cryodrgn analyze` at the end.

### ISSUE THREAD

**Issue by adamlerer** [zhonge/cryodrgn#283]

**Comment by Guillawme** [comment]
I agree it's a good default behavior to automatically analyze the last iteration.

But it would be nice if this could be turned off too. When using a cluster and a queuing system, one might want to request different resources for analysis than for training.

**Comment by michal-g** [comment]
fyi, we are working on a version of this behaviour (auto analysis of the final epoch at the end of training, with a flag for train-only) for an upcoming release!
